blazor/datagrid/selection.md

```diff
@@ -253,6 +253,54 @@ and hold CTRL key and click the desired rows. To select range of rows, press and
 }
 ```
 
+### How to hide select all checkbox in checkbox column
```

Comment:
Change the title as Hide select all checkbox in header

Reply:
title changed. resolved